### ISSUE TITLE
The plugin does not work

### ISSUE THREAD

**Issue by congthien** [PepperPlatform/yourls-params-append#1]
I added info to platform name & URL param, but it does not save into database: https://cl.ly/efa8d580488c

Could you check it please?

**Comment by tm57** [comment]
did you dump init.sql the to you database? You have to make sure the `init.sql` query is run against your database

**Comment by congthien** [comment]
> did you dump init.sql the to you database? You have to make sure the `init.sql` query is run against your database

Yes. I imported that file to my database. Please see: https://cl.ly/e0af968956ae

**Comment by tm57** [comment]
Right, which version of YOURLS are you using? Is it also `1.7.3`

**Comment by congthien** [comment]
> Right, which version of YOURLS are you using? Is it also `1.7.3`

Thank you. It works. However, I have a problem.
For example I added Platform is **Facebook**, url params is **fbclid**

but then the url will show like this: https://domain.com/?fbclid&utm_campaign=12121212121
It always has utm_campaign after URL. Can you let me know why?

**Comment by congthien** [comment]
Hi @tm57 

Do you have any clue?
